src/web_app_skeleton/config/logger.cr

```diff
@@ -38,7 +38,7 @@ Lucky::LogHandler.configure do |settings|
   if Lucky::Env.development?
     settings.skip_if = ->(context : HTTP::Server::Context) {
       context.request.method.downcase == "get" &&
-      context.request.resource.starts_with?(/\/css\/|\/js\/|\/assets\//)
+      context.request.resource.starts_with?(/\/css\/|\/js\/|\/assets\/|\/favicon\.ico\//)
```

Comment:
This wouldn't match `/favicon.ico` https://play.crystal-lang.org/#/r/7u9a That last slash should go away.

Reply:
🤦‍♂ Now that's embarrassing. Thanks for catching this! I've checked that it works in logs, but later I did some other experiments and it ended up like this.

Reply:
no worries!